### PR TITLE
signIn done / refactore nav-app routes done

### PR DESCRIPTION
**Pull request by Sky-De**
hi there 
signIn is done with exact match with design for desktop 
what i did :
1.image for signIn page added to assests
2.signIn page done / accourate 
3.routes moved from navbar to app file for cleaner and more maintainable code
4.NavLink added to navbar and active class added to index.css to handle active link

result DESKTOP :

desktop

result MOBILE:
mobile

If you got any question feel free to ask 
otherwise merge this 
thanks

### CODE REVIEW

**Review by mauricenyanja — CHANGES_REQUESTED** [review]
1. Dont mix issues
2. Use react-icons 
3. Remove the images as icons
4. Use tailwind css

File: src/pages/SignIn.jsx
[the remark this answers is not in the document]
Comment (Sky-De) [reply src/pages/SignIn.jsx:1]: Hi @mauricenyanja 

1. I didn't mix issues, i just created signIn and another thing was just moving routes from nav to app file which is standard method

2. I've used tailwind for styles and only part which is not tailwind is realated to .active class which is needed to add style to selected NavLink ( take a look to difference between Link and NavLink )
3. At the very first comment that i sended to ask to get assigned to this issue i asked you about assets and you replayed as : ** Hello 👋 @Sky-De pick it and there is no image just create yours** so i did

4.in your note just number 2 is something we can talk about, for this specific change

At last will you merge this if i replace images with icons from react-icons?

[the remark this answers is not in the document]
Comment (mauricenyanja) [reply src/pages/SignIn.jsx:1]: oakay I have seen sign in and navbar those are two different issues ama?
Plus U haven't seen u linking issues so what are u solving this is how we do ie #27

[the remark this answers is not in the document]
Comment (mauricenyanja) [reply src/pages/SignIn.jsx:1]: Remove the nav and sign in separate them 
Then use react icons for the icons

**Comment by mauricenyanja** [comment]
redo it and send a separate pr
Maurice Emmanuel Nyanja
SoftwareCloud EngineeringFreelance

   - [image: Phone icon] Phone: +254706705151
   - [image: Mobile icon] Mobile: +254700427697
   - [image: Email icon] Email: ***@***.***
   - [image: Website icon] Website: https://maurice-nyanja.netlify.app/
   - [image: Address icon] Address: Westlands, Coop


   - [image: twitter icon] <https://twitter.com/cheatcodesdev>
   - [image: linkedIn icon]
   <https://www.linkedin.com/in/maurice-nyanja-b84458202/>

**Pull request closed without merging**